kore/src/Kore/Options.hs

```diff
@@ -77,10 +77,12 @@ data KoreParserOptions = KoreParserOptions
 
 parsePatternOptions :: Parser PatternOptions
 parsePatternOptions = PatternOptions
-    <$> strOption
-        (  metavar "PATTERN_FILE"
-        <> long "pattern"
-        <> help "Kore pattern source file to parse [and verify]. Needs --module.")
+    <$> some (
```

Comment:
Use `many` here; `some` requires at least one `--pattern`, but `kore-parser` can be used to verify a definition and no patterns.

```suggestion
    <$> many (
```

Reply:
> Use `many` here; `some` requires at least one `--pattern`, but `kore-parser` can be used to verify a definition and no patterns.

This `some` is nested inside the optional `parsePatternOptions`. So what this is basically saying is if you want to have the patterns/module options combo then you must specify at least one pattern.